src/tvheadend/HTSPVFS.cpp

```diff
@@ -32,6 +32,8 @@ extern "C"
 #include "Settings.h"
 #include "utilities/Logger.h"
 
+#include <ctime>
```

Comment:
Well, I meant to put it after `#include "libhts/htsmsg_binary.h"` (but outside the extern C block), please.

Reply:
Done